lametro/templates/lametro/person.html

```diff
@@ -83,20 +87,33 @@ <h4>
                     </p>
 
                     <p class="small"><a href="/about/#about-la-metro"><i class="fa fa-info-circle" aria-hidden="true"></i> More about Metro appointments</a> </p>
+                    {% if user.is_authenticated %}
+                    {% include 'partials/person_headshot_form.html' %}
+                    {% endif %}
                 </div>
             </div>
 
             <hr/>
         </div>
 
         <div class='col-sm-8 no-pad-mobile'>
-            {% if member_bio %}
+            {% if member_bio or person.biography %}
```

Comment:
Similarly to `member.headshot_url`, how about adding a property to the LAMetroPerson model that conditionally retrieves `person.biography`, falling back to the baked in biographies from the settings?